src/main/java/org/sagebionetworks/web/client/widget/discussion/DiscussionThreadWidget.java

```diff
@@ -1,23 +1,29 @@
 package org.sagebionetworks.web.client.widget.discussion;
 
 import org.sagebionetworks.repo.model.discussion.DiscussionThreadBundle;
+import org.sagebionetworks.web.client.GWTWrapper;
 import org.sagebionetworks.web.client.PortalGinInjector;
 
 import com.google.gwt.user.client.ui.Widget;
 import com.google.inject.Inject;
 
-public class ThreadWidget implements ThreadWidgetView.Presenter{
+public class DiscussionThreadWidget implements DiscussionThreadWidgetView.Presenter{
 
-	ThreadWidgetView view;
+	DiscussionThreadWidgetView view;
 	PortalGinInjector ginInjector;
+	GWTWrapper gwtWrapper;
+	boolean areRepliesConfigure;
```

Comment:
When a user click on DiscussionTab, the first 10 thread in the project is loaded. However, it's replies are not loaded. The first time, user click on "x replies" on a thread, the first n number of replies for that thread id loaded. However, clicking on the same element "x replies" from that point on, only the toggle functionality is triggered. We will not load the first n replies again. So I added a check to make sure I only load them and add them once. To load more replies from that point, user will use "load more" button.